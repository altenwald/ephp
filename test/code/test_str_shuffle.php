<?php

var_dump(str_shuffle('axbycz'));
var_dump(str_shuffle(''));
var_dump(str_shuffle(NULL));
