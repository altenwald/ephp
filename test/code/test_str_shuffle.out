string(6) "abcxyz"
string(0) ""
string(0) ""
